### ISSUE TITLE
Meta-bug for task ownership

### ISSUE THREAD

**Issue by addyosmani** [yeoman/yeoman#997]
Hey team,

**tl;dr: if you have time, could you pick an area on the list below you can commit to owning?**

First, I wanted to thank you all for your fantastic work helping answer issues and file patches against the project and related projects like Grunt and Bower.

As we go into Q2 of 2013, there are a few key areas that Yeoman could really improve on and we would love your help to achieve the following:
- [ ] Finalize outstanding issues on Windows support
- [ ] Document a recommended install path for Windows users in the wiki
- [ ] Port Grunt init templates to yo 1.0
- [ ] Lopri: Update Ember generator to 1.0
- [ ] Lopri: Update Backbone generator to 1.0
- [ ] Improve readmes for existing generators
- [ ] Docs: help with one of our [existing docs asks](https://github.com/yeoman/yeoman/issues/829)
- [ ] Docs: RequireJS workflow
- [ ] Docs: Jade/Less workflow
- [ ] Tutorials: common development workflows with Yo/Bower/Grunt
- [ ] Tutorials: building simple apps with our workflow
- [ ] Site: help keep it up to date and make sure it captures docs in wiki
- [ ] Site: screencasts. We will need an 'official' one (Paul?) and less formal ones showing the general workflow.
- [ ] Outreach: speak to external generator authors and help them update their generators to 1.0
- [ ] Outreach: monitor questions about Yeoman on SO, answer where possible or link back to the docs.

I will personally be taking ownership of the generator system and writing new docs for it. For many of these items we should have an existing ticket for it, if not, please create one if you are taking the item on.

Now that we are modular, we need owners for the above in order to continue helping improve developers with their workflow.

I hope you can help us keep the project rockin :)

cc @passy @sindresorhus @sleeper @kevva @paulirish @btford
